database/src/main/java/entities/Genre.java

```diff
@@ -1,25 +1,28 @@
-package entities;
-
-import lombok.NoArgsConstructor;
-
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import java.io.Serializable;
-
-@Entity
-@NoArgsConstructor
-public class Genre implements Serializable {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private long id;
-
-    private String name;
-
-    public Genre(String name) {
-        this.name = name;
-    }
-
-}
+package entities;
+
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.io.Serializable;
+
+@Entity
+@NoArgsConstructor
+public class Genre implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String name;
+
+    public Genre(String name) {
+        this.name = name;
+    }
+
+    public String getName() {

```

Comment:
I have some problems with hibernate after changing to final, so I'll stay with that version.